<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>486</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>486</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(23, 33, 43);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1px solid rgb(36, 47, 61);
border-top: 1px solid rgb(36, 47, 61);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="Dir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border: 1px solid &quot;#30C193&quot;;
border-radius: 5px;
color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;</string>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>File or Directory path</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-radius: 5px;
background-color: rgb(36, 47, 61);
color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border: 1px solid &quot;#30C193&quot;;
border-radius: 5px;
color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generatepass">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-radius: 5px;
background-color: rgb(36, 47, 61);
color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="base.qrc">
       <normaloff>:/prefix1/icon/refresh.png</normaloff>:/prefix1/icon/refresh.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>266</x>
       <y>64</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="replacepassw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border: 1px solid &quot;#30C193&quot;;
border-radius: 5px;
color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>266</x>
       <y>109</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="hideorseek">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-radius: 5px;
background-color: rgb(36, 47, 61);
color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="base.qrc">
       <normaloff>:/prefix1/icon/eye.png</normaloff>:/prefix1/icon/eye.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1px solid rgb(36, 47, 61);
border-top: 1px solid rgb(36, 47, 61);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="aes256radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;
border: none;
</string>
     </property>
     <property name="text">
      <string>AES256</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1px solid rgb(36, 47, 61);
border-top: 1px solid rgb(36, 47, 61);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: &quot;#30C193&quot;;
font: 700 13pt &quot;Corbel&quot;;
border: none;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#encryptButton {
border: none;
color: &quot;#30C193&quot;;
font: 700 10pt &quot;Corbel&quot;;
border-radius: 5px;
}
#encryptButton:hover {
border: 1px solid &quot;#30C193&quot;;
}</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#decryptButton {
border: none;
color: &quot;#30C193&quot;;
font: 700 10pt &quot;Corbel&quot;;
border-radius: 5px;
}
#decryptButton:hover {
border: 1px solid &quot;#30C193&quot;;
}</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="base.qrc"/>
 </resources>
 <connections/>
</ui>
